src/bcf/header.rs

```diff
@@ -296,7 +296,7 @@ impl HeaderView {
         tag: &[u8],
         hdr_type: ::libc::c_uint,
     ) -> Result<(TagType, TagLength), TagTypeError> {
-        let (_type, length) = unsafe {
```

Comment:
Out of curiosity, what is the benefit of `_` for the `type` variable in this context?

Reply:
I think it is unused, and hence we silence the warning of allocating an unused variable here.

Reply:
It seems to be used since it is returned by the function, isn't it?

Reply:
Ah, `type` is of course a keyword in rust - that explains it.